### ISSUE TITLE
Does the image of WCAG1.4.11 'Dynamic Examples' satisfy WCAG 1.4.11?

### ISSUE THREAD

**Issue by sukoyakarizumu** [w3c/wcag#1670]
I'm not sure that the image of [WCAG 1.4.11 'Dynamic Examples'](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html#dynamic-examples) is satisfy WCAG 1.4.11 or not.

The understanding says:

> In order for someone to discern the graphics exist at all, there must be contrasting colors or text in order to find the graphics.

So I think we have to discern each slice of the pie chart from white background.
One of slice of the chart's color is pale purple(rgb(212, 199, 238)), and it's contrast ratio is insufficient.

Does the pie chart satisfy WCAG 1.4.11?

**Comment by JAWS-test** [comment]
The example is strange, because nowhere in the SC text does it say that it is sufficient to meet the contrast requirements only with focus or mouse over. And if this is so: Does this also apply to text (1.4.3)? Especially since the graphic also contains text. So would a page with a contrast of 1.5:1 for all text content be conform, if I can hover with the mouse over each paragraph and then it is displayed with sufficient contrast? I don't think this is the intent of 1.4.3 - and thus shouldn't be of 1.4.11 either

**Comment by JAWS-test** [comment]
> In order for someone to discern the graphics exist at all, there must be contrasting colors or text in order to find the graphics

It is currently not possible to judge whether figure 10A currently fulfills the wording of the Understanding, because there is only a screenshot in hover state. I.e. it is not possible to see how the graphic looks without focus or hover. Maybe there is an element with sufficient contrasts, maybe not. In this respect it would be better to implement a real example with interaction possibility or 2 images (without hover and with hover).

**Comment by detlevhfischer** [comment]
I think it is not only on hover or focus that contrast is to be met. My understanding is that there must be a default which conforms (here it is the blue segment with while text (4.7:1), and the black on white text in the legend). This is now encapsulated in the text that says:" In order for someone to discern the graphics exist at all, there must be contrasting colors or text in order to find the graphics."  Interacting with the graphic will make some other segment conform (and the blue one presumably fade).  Maybe the fact that one segment always has to have >4.5:1 contrast needs to be emphasized more.

**Comment by bruce-usab** [comment]
This bit (emphasis added) <q>for someone to discern the graphics **_exist at all_**</q> is a bit hyperbolic, no?  The color of the pie wedges would have be quite a bit more faint for anyone to (visually) confuse the graphic as white space.  Someone with a severe visual impairment (but not necessarily using screen reading software) will notice that there is _something_ but cannot make any sense out of it (i.e., cannot discern it).    I think this is a good technique.  Moreover, the contrast of the objects being ambiguous is a feature IMHO.  Sure, it is an example where the pie wedges could also have black (or white) borders, but the point is to have a barrier, and illustrate a sufficient technique for remediation.

**Comment by sukoyakarizumu** [comment]
@JAWS-test @detlevhfischer @bruce-usab 
Thank for your many comments.
I think the reason I was confused was that I couldn't imagine the initial state. (I misunderstood that all segments are pale color as initial state.)
Thanks to your comments and #1672, I understand what this sentence is trying to convey.
